etc/leapp/transaction/to_remove

```diff
@@ -1,51 +1,2 @@
-redhat-rpm-config
-python2-devel
```

Comment:
If I am right, this one was added because of conflicting obsoletes/provides, not because of rich-deps - or at least the issue with provides/obsoletes has been discovered later. Maybe it was a different problem, but rather check that python2-devel is not problematic from this point.

More context:
- python-devel on RHEL 7 provides python2-devel so we cannot install python2-devel rpm because:
```
 # dnf install python2-devel
   Package python-devel-2.7.5-76.el7.x86_64 is already installed. 
```

Reply:
I went by git blame which said otherwise